src/engine/strat_engine/engine.rs

```diff
@@ -418,6 +418,12 @@ impl Engine for StratEngine {
                 ));
             }
 
+            let block_size_summary = unowned_devices.blocksizes();
+            if block_size_summary.len() > 1 {
+                let err_str = "The devices specified for initializing the pool do not have uniform physical and logical sector sizes.".into();
+                return Err(StratisError::Msg(err_str));
+            }
```

Comment:
Maybe this is better to discuss in stand up, but what is the danger of having a different logical and physical block sizes? Is this the torn write problem?

Reply:
Perhaps I should reword that? It is totally ok for the physical sizes to differ from the logical sizes, and we would always allow that. How about: "Some of the devices specialized for initializing the pool have different logical sectors sizes or have different physical sector sizes." ?